### PR TITLE
Updated examples/overfitting/softmax.ipynb for 1.0

### PR DESCRIPTION
**Pull request by OsmanMutlu**

### CODE REVIEW

**Comment by denizyuret** [comment]
Thank you.  Merging this manually into Knet/tutorial in dy/1.1 branch.
